### DISCUSSION TITLE
Failed to start flyte-sandbox docker image error

### DISCUSSION THREAD

**Discussion by FlowerSaid** [flyteorg/flyte#6011] [Q&A]
Use command “docker run -d -p 30081:30081 -p 30080:30080 --name flyte-sandbox ghcr.io/flyteorg/flyte-sandbox:latest” to start the Docker container, and the log inside the container shows an error message: “Error: chart "flyte" matching v0.19.2-b5 not found in flyteorg index. (try 'helm repo update'): no chart version found for flyte-v0.19.2-b5”. What's the reason for this?How should I start it？

**Comment by Future-Outlier** [comment]
you can use
```zsh
flytectl demo start --image ghcr.io/flyteorg/flyte-sandbox:latest
```
